### PR TITLE
Ft comments#175743328

### PR DESCRIPTION
**Pull request by ConstantineMaliza**
#### What does this PR do?
- Comment thread

#### Description of Task to be completed?
- User is able to post a comment to a travel request to give additional information
- User is able to see the posted comment within the thread
#### How should this be manually tested?

- Clone the repository
- In your home directory cd this branch ft comments
- Then run npm install to install all dependencies
- Run npm start to start the server
- In you postman go to http://localhost:5000/requestId/comment to write a comment **for now, you can use any id** 
- To read comment use endpoint `GET` http://localhost:5000/requestId comments
- `All endpoint you get after sign in or sign out`

#### Any background context you want to provide?
N/A
What are the relevant Pivotal Tracker stories?
#175743328
Screenshots (if appropriate)
N/A
Questions:
N/A

### CODE REVIEW

File: src/routes/comment.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+
```
Comment (SamuelSabasNayo) [inline src/routes/comment.js:1]: Please remove this empty line.

**Review by SamuelSabasNayo — CHANGES_REQUESTED** [review]

File: src/routes/comment.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+ *          required: true
+ * 
+ *      responses:
+ *        "200":
+ *          description: Receive Comments
+ *        "500":
+ *          description: Error  
+ */
+
+router.get('/:requestId/comments', auth, GetComment);
```
Comment (SamuelSabasNayo) [inline src/routes/comment.js]: About this **url** format, can anyone see a comment of an imagined request? or you have to see the request and related comments.

**Comment by SamuelSabasNayo** [comment]
Again, resolve conflicts in app.js.

File: src/routes/comment.js (hunk 2 above)
Comment (ConstantineMaliza) [reply src/routes/comment.js]: yes ResquestId for now is  imagined because I don't have the real request

Comment (ConstantineMaliza) [reply src/routes/comment.js]: Yes that why need **travel request for jemima'work**

**Pull request closed without merging**